### ISSUE TITLE
Refreshing on Translations tab causes Desk tool to crash

### ISSUE THREAD

**Issue by apennell** [sanity-io/sanity-translations-tab#4]
## tl;dr

When I am in the Translations tab on a document and refresh the page, the Desk tool crashes when attempting to reload, because `displayed._id is undefined`. If I navigate away from this document+tab url, and then make my way back to the translations tab by clicking through the documents in the UI, it then loads fine.

![Screen Shot 2021-11-10 at 2 01 27 PM](https://user-images.githubusercontent.com/11039888/141204636-ca10c23e-0f2f-4fbc-b38e-39e329c35f43.png)

## Steps to reproduce

1. Implement `TranslationsTab`--my case was following the basic setup of [`sanity-plugin-studio-smartling`](https://github.com/sanity-io/sanity-plugin-studio-smartling) using the `defaultFieldLevelConfig`, which implements `TranslationsTab` from this repo under the hood:
    ```
    import S from '@sanity/desk-tool/structure-builder'
    import { TranslationsTab, defaultFieldLevelConfig } from 'sanity-plugin-studio-smartling'

    export const getDefaultDocumentNode = (props) => {
      return S.document().views([
        S.view.form(),
        S.view.component(TranslationsTab).title('Smartling').options(
          defaultFieldLevelConfig
        )
      ])
    }
    
    export default () =>
      S.list()
        .title('Content Type')
        .items(
          S.documentTypeListItems()
        )
    ```
2. Click through the desk structure to a document that has the Translations tab, then click on that tab. Content loads fine.
3. Refresh the browser page
4. Tries to load but Desk tool crashes with error `displayed._id is undefined`, where `displayed` is the document.
5. Click "Desk" and refresh to get back to the successfully loaded desk, then click through to the document and Translations tab again to see that it does successfully load

## System versions used

@sanity/core 2.21.11, all other @sanity packages 2.21.10
Firefox 94.0.1

---

Thank you for your hard work on great tools! 👏

**Comment by apennell** [comment]
I just double-checked this behavior in my studio because I was pretty sure it had been fixed by #5 but now I'm seeing a sneaky new bug. If I refresh, it loads one language's progress bar, rather than the correct job progress bars for the selected document. In some cases, it should have shown no jobs because I'd never created one; in another, I had jobs created for all languages. If I'm clicking through as normal (no refreshing, just navigating by clicking between docs and their view and translation tabs), I see the correct job progress bars.

I'm thinking it's an issue with incorrectly stored context, because it's the one language that I would have created a job for when I was last testing the translation connection in the plugin-studio-smartling, but haven't investigated much beyond confirming the behavior. Happy to provide any additional info on this.

**Comment by cngonzalez** [comment]
Closing because of age and it has not been reported since. Please re-open if you see similar behavior!
